Source/WTF/wtf/glib/GSpanExtras.h

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright (C) 2024 Igalia S.L.
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Library General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Library General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Library General Public License
+ *  along with this library; see the file COPYING.LIB.  If not, write to
+ *  the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ *  Boston, MA 02110-1301, USA.
+ */
+
+#pragma once
+
+#include <wtf/StdLibExtras.h>
+#include <wtf/glib/GRefPtr.h>
+
+namespace WTF {
+
+inline std::span<const uint8_t> span(GBytes* bytes)
+{
+    std::size_t size = 0;
```

Comment:
I think one of our unwritten code style guidelines is to not write `std::` for stuff inherited from C:

```suggestion
    size_t size = 0;
```